VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls-base.component.spec.ts

```diff
@@ -611,10 +604,16 @@ describe('HearingControlsBaseComponent', () => {
         expect(component.isJOHConsultation).toBe(true);
     });
 
-    it(`canShowScreenShareButton() should return "false" when device is a tablet`, () => {
-        deviceTypeService.isTablet.and.returnValue(true);
+    it(`canShowScreenShareButton() should return "false" when device is not desktop`, () => {
```

Comment:
These tests should be grouped inside a describe block instead of being prefixed by "canShowScreenShareButton()" each time.

`describe('canShowScreenShareButton'), () => {
    it(`should return "false" when device is not desktop`, () => {
}`

It reads cleaner that way, is easier to maintain and allows you to do setup specific to testing that method if needed.